src/routes/docs/products/sites/quick-start/vue/+page.markdoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: article
+title: Deploy a Vue.js app to Appwrite Sites
+description: Learn how to setup and deploy Vue.js apps on Appwrite Sites.
+---
+
+{% section #step-1 step=1 title="Create Vue.js app" %}
+
+{% info title="Important note" %}
+You can skip this step if you have developed a Vue.js app already.
+
+If you prefer, you can also directly setup our [Vue.js starter template](/docs/products/sites/templates) instead.
+{% /info %}
+
+Open your terminal, and run the following command.
```

Comment:
Using a card for this here takes too much focus, like using 5kg hammer to hang a nail on the wall. Feels a bit negative to start a quick start with what looks like a warning. Instead, maybe we can just include it in the sentence.

Reply:
Simplified to a single sentence